File: linguagensComerciais/python/estruturaCondicionalSimplesEComposta.py
idade = 10
# criança, adolescente, adulto ou idoso

""""
if idade >= 0 and idade < 12:
    print("Criança")

print("Final")

# estrutura condicional composta
if idade >= 0 and idade < 12:
    print("Criança")
else:
    print("Adulto")

print("Final")
"""

if idade < 0:
    print("Valor inválido")
elif idade >= 0 and idade < 12:
    print("Criança")
elif idade < 18:
    print("Adolescente")
elif idade < 60:
    print("Adulto")
else:
     print("Idoso")

print("Final")

File: linguagensComerciais/javascript/estruturaCondicionalSimplesEComposta.js
// criança, adolescente, adulto ou idoso

var idade = 80

// estrutura condicional simples
/*
if (idade >= 0 && idade <= 12) {
    console.log("Criança")
} 
*/

// estrutura condicional composta
/*
if (idade >= 0 && idade <= 12) {
    console.log("Criança")
} else {
    console.log("Adulto")
}
*/

// estrutura condicional composta com condições encadeadas
if (idade >= 0 && idade <= 12) {
    console.log("Criança")
} else if (idade > 12 && idade < 18) {
    console.log("Adolescente")
} else if (idade >= 18 && idade < 60) {
    console.log("Adulto")
} else {
    if (idade < 0 || idade > 120) {
        console.log("Valor inválido")
    } else {
        console.log("Idoso")
    }
}